installation.rst

```diff
@@ -233,8 +233,18 @@ Juju will then download the charms from Charmhub, create the applications,
 configurations, and LXD containers described in the respective files, which
 will comprise the model.
 
+.. note::
+
+   The slurm-charms install SLURM from `Omnivector's OSD PPA
+   <https://launchpad.net/~omnivector/+archive/ubuntu/osd>`_ on Ubuntu. It is
+   possible to change the source to `Omnivector's Testing PPA
+   <https://launchpad.net/~omnivector/+archive/ubuntu/osd-testing>`_ (or to a
+   local cache server as well) whith the configuration ``custom-slurm-repo``.
+   Setting this value to repositories other than Omnivector's PPAs is not
```

Comment:
Might be good to make this part bold or highlight it a bit

Reply:
It already is! Just pushed to `testing`: https://omnivector-solutions.github.io/osd-documentation/testing/installation.html#deploy-slurm